x/concentrated-liquidity/position_test.go

```diff
@@ -2440,31 +2440,34 @@ func (s *KeeperTestSuite) TestTransferPositions() {
 					s.Require().Equal(oldPosition, newPosition)
 				}
 
-				// Check that the incentives and spread rewards went to the old owner
+				// Check that the old owner's balance did not change due to the transfer
 				postTransferOriginalOwnerFunds := s.App.BankKeeper.GetAllBalances(s.Ctx, oldOwner)
-				expectedTransferOriginalOwnerFunds := preTransferOwnerFunds.Add(totalExpectedRewards...)
-				s.Require().Equal(expectedTransferOriginalOwnerFunds.String(), postTransferOriginalOwnerFunds.String())
+				s.Require().Equal(preTransferOwnerFunds.String(), postTransferOriginalOwnerFunds.String())
 
-				// Check that the new owner does not have any new funds
+				// Check that the new owner's balance did not change due to the transfer
 				postTransferNewOwnerFunds := s.App.BankKeeper.GetAllBalances(s.Ctx, newOwner)
 				s.Require().Equal(preTransferNewOwnerFunds, postTransferNewOwnerFunds)
 
-				// Test that claiming incentives/spread rewards with the new owner returns nothing
```

Comment:
Seems like this diff is rendering weirdly. The changes here were:
1. Remove previous assertion that claiming rewards post transfer yields nothing
2. Claim rewards post transfer and ensure they go to new owner